AirCasting/Utils/DataFormatters.swift

```diff
@@ -112,7 +112,8 @@ enum DateFormatters {
         static let fileParserFormatter: DateFormatter = {
             let dateFormatter = DateFormatter()
             dateFormatter.locale = Locale(identifier: "en_US_POSIX")
-            dateFormatter.dateFormat = "MM/dd/yyy'T'HH:mm:ss"
+            // yyy?
```

Comment:
What is the purpose of this?

Reply:
it's a leftover. There was yyy and I changed it to yyyy but earlier left a comment as a remainder to ask Ania about it :D